tuf/keys.py

```diff
@@ -554,7 +573,7 @@ def check_crypto_libraries(required_libraries):
   tuf.formats.REQUIRED_LIBRARIES_SCHEMA.check_match(required_libraries)
  
   # The checks below all raise 'tuf.UnsupportedLibraryError' if the general,
```

Comment:
I would probably move the checks below into their own function. (That said, I'm not sure why this is the place to make sure they're available in the first place, so maybe I'm missing something.)